api/unstable/minimal_config.h

```diff
@@ -0,0 +1,49 @@
+/*
+* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+#pragma once
+
+#include <s2n.h>
+
+/**
+ * Creates a new s2n_config object with minimal default options.
+ *
+ * This API is similar to `s2n_config_new`, except that the returned config is created without
+ * loading system certificates into the trust store. To add system certificates to this config,
+ * call `s2n_config_load_system_certs`.
+ *
+ * The returned config should be freed with `s2n_config_free` after it's no longer in use by any
+ * connections.
+ *
+ * @returns A new s2n_config object
+ */
+S2N_API struct s2n_config* s2n_minimal_config_new(void);
+
```

Comment:
This name probably needs to start with "s2n_config", whatever else we choose. So like "s2n_config_new_minimal" or "s2n_config_minimal_new".

Reply:
Right, that makes sense. I changed it to `s2n_config_new_minimal`.